src/advertising.ts

```diff
@@ -2,17 +2,35 @@
  * Quartiles that can be reached during an ad playback.
  */
 export enum AdQuartile {
+  /**
+   * Fist ad quartile.
+   */
   FIRST = 'first',
+  /**
+   * Mid ad quartile.
+   */
   MID_POINT = 'mid_point',
+  /**
+   * Third ad quartile.
+   */
   THIRD = 'third',
 }
 
 /**
  * The possible types an `AdSource` can be.
  */
 export enum AdSourceType {
+  /**
+   * Google Interactive Media Ads.
+   */
   IMA = 'ima',
+  /**
+   * Unknown advertising source type.
+   */
   UNKNOWN = 'unknown',
+  /**
+   * Progressive ad type.
+   */
```

Comment:
I would prefer a consistent naming: `ad type` vs. `advertising source type`.

Reply:
🤦 copied from the iOS SDK 🙈 

@strangesource which one would you prefer?

Reply:
`advertising source type`, but no strong opinion. :)

Reply:
I went for `ad source type` because we had no other occurrence of `advertising` but `ad` was used everywhere in current API documentation. So consistency requires us to use `ad`.

See: 8507fa0